src/models/Token.ts

```diff
@@ -1,23 +1,32 @@
-import mongoose from 'mongoose';
+import mongoose, { Schema } from 'mongoose';
 import { DateTime } from 'luxon';
 import { Service } from 'typedi';
 
-
-const roleSchema = new mongoose.Schema({
-  roleName: {
+const tokenSchema = new mongoose.Schema({
+  token: {
     type: String,
     required: true
+  },
+  user_id: {
```

Comment:
I think userId is better because it's more consistent with other properties naming convention